cmd.proto

```
@@ -17,6 +17,7 @@ message Response {
         string v_str = 4;
         double v_float = 5;
         bytes v_bytes = 6;
+        repeated string v_slice = 7;
```

Buf (1.47.2)

syntax error: unexpected "repeated"

(COMPILE)